### DISCUSSION TITLE
'rounded' since it isn't a known property of 'p-button' after update to 16.7.2

### DISCUSSION THREAD

**Discussion by aaron-za-dev** [primefaces/community#485] [PrimeNG] [answered]
I am trying to use the rounded variant as shown in the documentation: 

<p-button icon="pi pi-bookmark" [rounded]="true" severity="secondary"></p-button>

But after updating to version 16.7.2 the following error is displayed:

**Can't bind to 'rounded' since it isn't a known property of 'p-button'.**

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Rounded property exists and it works. Maybe you're missing an import?

https://stackblitz.com/edit/vzmbw5?file=src%2Fapp%2Fdemo%2Fbutton-rounded-demo.html
